config/job.yaml.tim

```diff
@@ -10,25 +10,32 @@ spec:
     metadata:
       labels:
         sdbuild: {{build_id}}
+      annotations:
+        pod.alpha.kubernetes.io/init-containers: '[
```

Comment:
Is this allowed to be here?

Seems weird that this looks like JSON format inside a yaml